src/lib/formatter/RuleResultRecombinator.ts

```diff
@@ -28,33 +28,46 @@ export class RuleResultRecombinator {
 				formattedResults = this.constructJunit(results);
 				break;
 			case OUTPUT_FORMAT.SARIF:
-				formattedResults = await constructSarif(results, executedEngines);
+				formattedResults = await constructSarif(results, executedEngines, outputOptions.normalizeSeverity);//
 				break;
 			case OUTPUT_FORMAT.TABLE:
 				formattedResults = this.constructTable(results);
```

Comment:
Would be helpful to have a code comment here explaining why constructTable() doesn't need the normalizeSeverity boolean.